Api/Services/EmploymentService.cs

```diff
@@ -136,4 +136,87 @@ public async Task<Result> UpdateBulkEmploymentAsync(List<UpdateEmploymentRequest
         await context.SaveChangesAsync();
         return Result.Success;
     }
+
+    /// <summary>
+    /// Gets current or completed employments for the user that makes this request
+    /// </summary>
+    /// <param name="userId">ID of the user that makes the request</param>
+    /// <param name="isTerminated">parameter that tells if the search should include terminated xor ongoing employments</param>
+    /// <returns></returns>
+    [ErrorCode(null, ErrorCodes.NotFound)]
+    public async Task<Result<List<EmploymentSummaryVM>>> GetCurrentUsersEmploymentsAsync(string userId, bool isTerminated)
+    {
+        var user = await context.Users.SingleOrDefaultAsync(u => u.Id == userId);
+        if (user is null)
+        {
+            return new ValidationFailure
+            {
+                PropertyName = null,
+                ErrorCode = ErrorCodes.NotFound,
+                ErrorMessage = ErrorCodes.NotFound
+            };
+        }
```

Comment:
Można dać po prostu SingleAsync, jeśli nie ma takiego usera, to to błąd aplikacji